packages/rest/src/rest.server.ts

```diff
@@ -614,7 +621,15 @@ export class RestServer extends Context implements Server, HttpServerLike {
    * @param options Options for serve-static
    */
   static(path: PathParams, rootDir: string, options?: ServeStaticOptions) {
-    this.httpHandler.registerStaticAssets(path, rootDir, options);
+    if (this._httpHandler) {
+      this.httpHandler.registerStaticAssets(path, rootDir, options);
+    } else {
+      this.staticAssets.push({
```

Comment:
I think you should always create a new entry in `this.staticAssets`, so that all static assets are preserved whenever the HTTP handler is rebuilt.

See #433 and 
https://github.com/strongloop/loopback-next/blob/1bcdb5b69221501b3952bd70c0c78409b600d1ab/packages/rest/src/rest.server.ts#L262-L267

I think this is the simplest version of this method:

```ts
this.staticAssets.push(/*...*/);
delete this._httpHandler;
```